Firestore/Source/Util/FSTDispatchQueue.mm

```diff
@@ -16,6 +16,8 @@
 
 #import <Foundation/Foundation.h>
 
+#import <atomic>
```

Comment:
I believe I just added [checks to cpplint](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/cpplint.py#L4545) to help with that and it has called you out: https://travis-ci.org/firebase/firebase-ios-sdk/jobs/365187148#L641 :-).

Reply:
Yes, I deleted the comment once I noticed, but you were quicker.